students/ZhuojingXie/lesson03/mailroom.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+import sys
+data=[['Andy',[960,256,123.5,40]],['Bryce',[30,45,27]],
+      ['Charile', [25,50]],['David', [10]],
+      ['Elaine',[75,26]]]
+# this is my data form
+
+
+def prompt():
+    user_select = input("\n\nPlease select a action: \n1 Send a Thank You!"
+                        "\n2 Create a Report  \n3 Quit\n\n  >")
+    while not user_select.isdigit():
+        user_select = input(" Please type a number")
+    while user_select not in ['1','2','3']:
+        user_select = input("\n\nPlease try again, select a action:\n1 Send a Thank You!"
+                            "\n2 Create a Report.  \n3 Quit\n\n  >")
+    return user_select
+# main menu
+
+
+
+def donor_name():
+    donor_list=[]
+    for info in data:
+        donor_list.append(info[0])
+    return donor_list
+#this function gonna return a list of donaor, and it will be used in thankprompt function later
+
+def thankprompt():
+    user_select_thank=input('\n\nType "list" to show a list of the donor names, \n'
+                            'Or type "back" to Back\n')
+    while user_select_thank.lower() != 'back' and user_select_thank.lower() != 'list':
+        user_select_thank=input('\n\nType "list" to show a list of the donor names, \n'
+                            'Or type "back to Back\n\n')
```

Comment:
similar to comment before to avoid duplication, you could have
```
user_select_thank = None
while user_select_thank not in ('back', 'list'):
    user_select_thank = input("....").lower()
```